README.md

```diff
@@ -1,170 +1,269 @@
-
-**Frontend Only**
+**Restart a single service:**
 
 ```bash
-docker-compose --profile frontend up --build
+docker-compose restart frontend
 ```
 
-Includes: Next.js frontend + MongoDB
-
-**Backend Only**
+**Rebuild after updates:**
 
 ```bash
-docker-compose --profile backend-local up --build
+docker-compose --profile local up --build
 ```
 
-Includes: FastAPI, Redis, Celery, Flower
+**Check running containers:**
 
-**AI Servers Only**
+```bash
+docker-compose ps
+```
+
+**Clean everything and start fresh:**
 
 ```bash
-docker-compose --profile servers up --build
+docker-compose down -v
+docker system prune -a
 ```
 
-Includes: infinite-talk-server, zonos-server (requires GPU)
+---
 
-**Everything**
+## Technology Stack
 
-```bash
-docker-compose --profile local --profile servers up --build
+- **Frontend**: Next.js, TypeScript, Tailwind CSS, Fabric.js
+- **Backend**: FastAPI, Python, UV package manager
+- **Database**: MongoDB
+- **Queue**: Celery with Redis
+- **Deployment**: Docker Compose
+
+### Project Structure
+
+```
+aafactory/
+├── frontend/          # Next.js application
+├── backend/           # FastAPI server
+├── docker-compose.yml # Service orchestration
+└── README.md         # This file
 ```
 
-Includes all services
+---
 
-## Development Commands
+## Troubleshooting
 
-### Stop all services
+**Port already in use?**
 
 ```bash
-docker-compose down
+# Find what's using the port
+lsof -i :3000
+# Kill it or change the port in docker-compose.yml
 ```
 
-### View logs
+**Container won't start?**
 
 ```bash
-docker-compose logs -f [service-name]
+# View detailed error logs
+docker-compose logs [service-name]
 ```
 
-### Rebuild specific profile
+**Need to reset everything?**
 
 ```bash
-docker-compose --profile [profile-name] up --build
+docker-compose down -v
+docker system prune -a
 ```
 
-## Architecture
+**Still stuck?**
+Join our [Discord](https://discord.gg/C2Rjy8Q2ER) for help!
 
-- **Frontend**: Next.js with TypeScript, Tailwind CSS, and Fabric.js for video editing
-- **Backend**: FastAPI with Python, using UV for dependency management
-- **Task Queue**: Celery with Redis for background processing
-- **Databases**: MongoDB for documents
-- **Containerization**: Docker Compose for local development
+---
 
-## Additional Resources
+## Contributing
 
-- [MongoDB Community Download](https://www.mongodb.com/try/download/community)
+We welcome contributions! Join our [Discord community](https://discord.gg/C2Rjy8Q2ER) to:
 
-## Notes
+- Ask questions and get support
+- Report bugs and suggest features
+- Share your avatars and projects
+- Collaborate with other developers
 
-The video editor component was initially cloned from: [fabric-video-editor](https://github.com/AmitDigga/fabric-video-editor/issues)
+---
 
-## Contributing
+## Credits
+
+Video editor component based on [fabric-video-editor](https://github.com/AmitDigga/fabric-video-editor).
+
+---
+
+## License
+
+[Your license here]
```

Comment:
Shall we remove the License part from here? As we have it in the License section of Github